src/components/content/content.style.js

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable import/prefer-default-export */
+import styled, { css } from "styled-components";
+import { baseTheme } from "../../style/themes";
+
+export const StyledContent = styled.div`
+  & + & {
+    margin-top: 15px;
+  }
+
+  text-align: ${({ align }) => align && align};
+`;
+
+export const StyledContentTitle = styled.div`
+  ${({ theme, titleWidth, inline, variant, align }) => {
+    return css`
+      display: ${inline ? "inline-block" : "block"};
+      font-weight: bold;
+      color: ${theme.text.colors};
+      width: ${titleWidth && `calc(${titleWidth}% - 30px)`};
+      text-align: ${!inline && align};
+
+      ${align === "center" &&
+      inline &&
+      css`
+        text-align: right;
+        width: calc(50% - 30px);
+      `}
+
+      ${variant === "secondary" &&
+      css`
+        color: ${theme.content.secondaryColor};
+        font-weight: normal;
+      `}
+    `;
+  }};
+`;
+
+export const StyledContentBody = styled.div`
+  ${({ bodyFullWidth, titleWidth, inline, align }) => {
+    return css`
+      display: ${inline ? "inline-block" : "block"};
+      margin-top: 2px;
+      white-space: pre-wrap;
+      word-wrap: break-word;
+      font-weight: normal;
+
+      ${bodyFullWidth &&
+      css`
+        width: 100%;
+      `}
+
+      ${titleWidth &&
+      css`
+        width: ${100 - Number(titleWidth)}%;
+      `}
+
+      ${inline &&
+      css`
+        margin-top: 0;
+        margin-left: 30px;
+        text-align: left;
+      `}
+
+      ${align === "center" &&
+      inline &&
+      css`
+        width: 50%;
+      `}
+    `;
+  }};
+`;
+
+StyledContent.defaultProps = { theme: baseTheme };
```

Comment:
`StyledContent` and `StyledContentBody` don't actually use the theme so you don't need these lines for those two